.travis.yml

```diff
@@ -1,18 +1,20 @@
+language: generic
+services:
+  - docker
 
-sudo: required 
-dist: trusty 
-language: generic 
-
+notifications:
+  email:
+    on_success: change
+    on_failure: always
 env:
+  global:
+    - ROS_REPO=ros
   matrix:
-    - ROS_DISTRO="indigo"  ROS_REPO=ros
-    - ROS_DISTRO="indigo"  ROS_REPO=ros-shadow-fixed
-    - ROS_DISTRO="kinetic" ROS_REPO=ros
-    - ROS_DISTRO="kinetic" ROS_REPO=ros-shadow-fixed
-
+    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO=melodic UPSTREAM_WORKSPACE=file
 install:
-  - git clone --depth=1 https://github.com/ros-industrial/industrial_ci.git .industrial_ci
-
-script: 
-  - .industrial_ci/travis.sh
-  
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
```

Comment:
Why did you change this?

Reply:
it's the `.travis.yml` we are using in all the other repos...i wanted to harmonize...
is something wrong about the new config?

Reply:
ah, now i see the `ros-shadow-fixed` being removed....should I re-add?

Reply:
now it's back to
 - adding melodic jobs
 - removing `sudo` and `dist`
 - adding `service`

I think this is in your interest now